src/core/Akka.Persistence.TestKit/PersistenceTestKit.cs

```diff
@@ -0,0 +1,35 @@
+//-----------------------------------------------------------------------
+// <copyright file="PersistenceTestKit.cs" company="Akka.NET Project">
+//     Copyright (C) 2009-2019 Lightbend Inc. <http://www.lightbend.com>
+//     Copyright (C) 2013-2019 .NET Foundation <https://github.com/akkadotnet/akka.net>
+// </copyright>
+//-----------------------------------------------------------------------
+
+namespace Akka.Persistence.TestKit
+{
+    using Actor;
+    using Akka.TestKit;
+    using Akka.TestKit.Xunit;
```

Comment:
For the benefit of other Akka.NET OSS users: we're taking an explicit dependency on XUnit here. This means that we're not going to be supporting NUnit as part of this feature. If this creates an issue for you, speak now or forever hold your peace.

Reply:
I will add a base class with assertions `ITestKitAssertions assertions` available for modification, but assembly will still reference `Akka.TestKit.Xunit2` unless there is 3rd assembly with integration.

Reply:
Sounds like a great compromise.

Reply:
`Akka.Persistence.TestKit.Xunit2`... but it will have just one class (`PersistenceTestKit`) 🤔